acceptance_tests/utilities/support_tool_healthcheck.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+from time import sleep
+
+from requests.exceptions import ConnectionError, HTTPError
+
+from acceptance_tests.utilities.iap_requests import make_request
+
+
+def support_tool_healthcheck(url: str, max_retries: int):
+    check_support_tool = True
+    num_of_retries = 0
+    while check_support_tool:
+        try:
+            response = make_request(url=url)
+            response.raise_for_status()
+            check_support_tool = False
+        except (ConnectionError, HTTPError):
+            num_of_retries += 1
+            if num_of_retries == max_retries:
+                print("Reached maximum number of retries. Stopping script")
+                break
```

Comment:
It looks like at the moment this will still exit with a success code and allow the tests to carry on, I think we would want it to fail out and abort the test attempt completely here?

Reply:
Just updated it not to exit instead of breaking. The task and run_gke script both should exit on failure. I've tested it with the run_gke script. The task looks similar so should do the same